app/utils/newsFeed/redis/mappings.ts

```diff
@@ -298,3 +305,24 @@ export const mapToRedisUser = async (userId: string): Promise<RedisUser> => {
   const user = await getInnoUserByProviderId(userId);
   return user;
 };
+
+const updateImageUrl = (imageUrl: string | undefined): string | undefined => {
```

Comment:
Update doesn't explain what the method does to the URLs, I'd suggest something like 'mapImageToRelativeUrl(...)'